create_package.py

```diff
@@ -253,79 +226,46 @@ def zip_client_side(addon_package_dir, current_dir, log):
 
     """
     client_dir = os.path.join(current_dir, "client")
+
     if not os.path.isdir(client_dir):
         log.info("Client directory was not found. Skipping")
         return
 
+    client_ayon_usd_download_dir = os.path.join(
+        current_dir, "client", "ayon_usd", "downloads"
+    )
+    shutil.rmtree(client_ayon_usd_download_dir)
```

Comment:
Unable to create package without this myself... unless I manually make the folder first?
```suggestion
    if os.path.exists(client_ayon_usd_download_dir):
        shutil.rmtree(client_ayon_usd_download_dir)
```

Reply:
Also, an empty downloads folder remains there... you may best best off picking a default `create_package.py` that @iLLiCiTiT has also bee recently deploying to other repos?

Reply:
Fixed with new create_package.py